Mixpanel/Persistence.swift

```diff
@@ -159,14 +164,16 @@ class Persistence {
         return data as? Queue ?? []
     }
 
-    class private func unarchiveProperties(token: String) -> (InternalProperties, InternalProperties, String, String?, Queue, Set<Int>) {
+    class private func unarchiveProperties(token: String) -> (InternalProperties, InternalProperties, String, String?, String?, Queue, Set<Int>) {
```

Comment:
Ouch... not sure what the appropriate style should be here to keep this line at 140 or less.